src/district/server/logging.cljs

```diff
@@ -108,7 +121,7 @@
   (merge data (decode-vargs (-> data
                                 :vargs))))
 
-(defn start [{:keys [:level :console? :file-path :sentry]}]
+(defn start [{:keys [:level :console? :file-path :file-roll-daily? :sentry]}]
```

Comment:
could we nest this option, since it applies only to the file log? ```{:file {:path "fu/bar" :roll-daily? true}}```

Reply:
yeah we can, I added the reason in the PR description, I did it this way so we don't break our users, not sure if we have any XD

Reply:
MF is the only place where we;re using it AFAIK (namebazaar did too, but is dead now so no concern here :P )